pkgs/servers/pulseaudio/default.nix

```diff
@@ -105,10 +117,15 @@ stdenv.mkDerivation rec {
     "-Dsysconfdir_install=${placeholder "out"}/etc"
     "-Dudevrulesdir=${placeholder "out"}/lib/udev/rules.d"
   ]
-    ++ lib.optional (stdenv.isLinux && useSystemd) "-Dsystemduserunitdir=${placeholder "out"}/lib/systemd/user"
-    ;
+  ++ lib.optional (stdenv.isLinux && useSystemd) "-Dsystemduserunitdir=${placeholder "out"}/lib/systemd/user"
+  ++ lib.optionals (stdenv.isDarwin) [
+    "-Ddbus=disabled"
+    "-Dglib=disabled"
+    "-Doss-output=disabled"
+  ];
 
-  doCheck = true;
+  # tests fail on Darwin because of timeouts
+  doCheck = if stdenv.isLinux then true else false;
```

Comment:
```suggestion
  doCheck = stdenv.isLinux;
```

Reply:
Done.

I disabled it on Darwin only because I think those tests may pass in other non-Darwin systems.